src/pages/VaccineAdminPage.js

```diff
@@ -34,13 +40,32 @@ const VaccineAdminPageComponent = ({
   onSortColumn,
   onToggleFridges,
   onToggleSensors,
+  onSelectLocation,
+  onEditSensorName,
+  onApplySensorLocation,
+  onCloseModal,
+  modalValue,
+  dispatch,
+  onAddFridge,
+  hasSelection,
+  onDeselectAll,
+  onDeleteRecords,
+  onCheck,
+  scanForSensors,
+  isScanning,
```

Comment:
whole bunch of callbacks, mainly reused `pages` actions with a couple more for sensor/location specific